bottles/frontend/ui/details-bottle.blp

```diff
@@ -230,6 +215,142 @@ template DetailsBottle : .AdwPreferencesPage {
 
   .AdwPreferencesGroup group_programs {
     title: _("Programs");
+
+    .AdwActionRow row_no_programs {
+      Box {
+        hexpand: true;
+
+        Label {
+          label: _("Use the Installers section or the \"Run executable\" button.");
+          wrap: false;
+
+          styles [
+            "dim-label",
+          ]
+        }
+      }
+    }
+    
+    .AdwPreferencesRow bottom_bar {
+      activatable: false;      
+      
+      Box {
+        height-request:45;
+        
+        Button add_shortcuts {
+          hexpand: true;
+          has-frame: false;
+          focusable: false;
```

Comment:
```        has-frame: false;
          focusable: false;```
          
these are unneccessary and break keyboard nav

Reply:
in my testing, `has-frame` is definitely required. `focusable` has been mistakenly pushed, i was experimenting with it earlier,  removed it :)

Reply:
2294c0ddcf2c4d75b9ae36b69247ae5fb6911abc